packit_service/worker/checker/testing_farm.py

```diff
@@ -125,3 +125,21 @@ def pre_check(self) -> bool:
             return False
 
         return True
+
+
+class IsIdentifierFromCommentMatching(Checker, GetTestingFarmJobHelperMixin):
+    """
+    Check that job identifier is matching comment --identifier option when it is specified.
+    If identifier is not specified it will allow all jobs execution,
+    otherwise only jobs with the same identifier.
+    """
+
+    def pre_check(self) -> bool:
+        if (
+            not self.testing_farm_job_helper.comment_arguments.identifier
+            or self.testing_farm_job_helper.comment_arguments.identifier
+            == self.job_config.identifier
+        ):
+            return True
+
```

Comment:
can we add logging here, something like "Skipping running tests for the job, identifiers don't match" and log the values as well?